### ISSUE TITLE
Feature Request : Create cluster as part of Install script

### ISSUE THREAD

**Issue by sunnygoel87** [cnoe-io/reference-implementation-aws#24]
Hi,

Currently cluster creation step is not part of install script. Is it possible to include that as a 1st item in the install script ? That way user doesn't need to run a separate command(`eksctl create cluster -f <filename>`) to provision the cluster. Everything required will be a part of install script.

**Comment by nimakaviani** [comment]
fair point. The initial idea for the reference implementation was to assume a bring-your-own-cluster setup, but we should be able to make it bootstrap itself.

**Comment by sunnygoel87** [comment]
> fair point. The initial idea for the reference implementation was to assume a bring-your-own-cluster setup, but we should be able to make it bootstrap itself.

Thanks @nimakaviani . Let me know if you want me to work on this.

**Comment by nimakaviani** [comment]
Thanks @sunnygoel87 .. we had a chat about this and landed on making this an optional feature for the script, eg., a flag that if passed to the script will run eksctl to create the cluster. if you are interested in taking this on, we definitely appreciate a PR :)
